final_project/snapvisite/models.py

```diff
@@ -112,7 +85,7 @@ def __str__(self):
 class Service(models.Model):
     name = models.CharField(max_length=128)
     description = models.TextField()
-    time = models.TimeField()
+    time = models.IntegerField(default=30, help_text="Put time in minutes. Like '60' = 1h, '30' = 30min, '90' = 1h 30min")
```

Comment:
Czemu time jest jako integerfield? To nie jest `time`, chodziło o `duration`? Jeśli tak to pytanie jak to się będzie miało do logiki związanej z umówienie spotkania na wykonanie usługi? Czyli połączenie tego z długością timeslota?